src/webots/launcher/launcher.c

```diff
@@ -51,46 +51,42 @@ static int fail(const char *function, const char *info) {
 }
 
 int main(int argc, char *argv[]) {
+  const wchar_t *original_command_line = GetCommandLineW();
+  wchar_t command_line[wcslen(original_command_line)];
+  const wchar_t *find = wcsstr(original_command_line, L"\\msys64\\mingw64\\bin\\webots");
+  int index = find - original_command_line;
+  index += 26;
+  wcsncpy(command_line, original_command_line, index);
+  command_line[index++] = '-';
+  command_line[index++] = 'b';
+  command_line[index++] = 'i';
+  command_line[index++] = 'n';
+  for (; original_command_line[index - 4]; index++)
+    command_line[index] = original_command_line[index - 3];
```

Comment:
I don't get exactly why here is `index - 3` instead of `index - 4`.
But in general it would be good to add some comments explaining what this piece of code is doing.